equinix/services/fabricv4/models/advance_configuration.py

```diff
@@ -78,9 +78,9 @@ def to_dict(self) -> Dict[str, Any]:
         # override the default output from pydantic by calling `to_dict()` of each item in ntp (list)
         _items = []
         if self.ntp:
-            for _item in self.ntp:
-                if _item:
-                    _items.append(_item.to_dict())
+            for _item_ntp in self.ntp:
+                if _item_ntp:
+                    _items.append(_item_ntp.to_dict())
```

Comment:
These sorts of changes are a result of https://github.com/OpenAPITools/openapi-generator/pull/19223, which fixes a potential bug when models contain multiple list attributes with different element types.